<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteClientClass</class>
 <widget class="QMainWindow" name="NoteClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>713</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>NoteClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Title :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="titleLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">backgroud-color: yellow;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectFileButton">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="codeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Showcode &gt;&gt;</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="fontsizeLabel">
        <property name="text">
         <string>Fontsize :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="fontsizeSpinBox">
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="fontsizeHorizontalSlider">
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="codeLabel">
      <property name="text">
       <string>Code :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QTextEdit" name="codeTextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>3</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="contentLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Content :</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QTextEdit" name="contentTextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="geneButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="clearButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="noteclient.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>codeButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>codeTextEdit</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>codeButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>NoteClientClass</receiver>
   <slot>setCodeButtonText(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>codeButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>codeLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontsizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fontsizeHorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontsizeHorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fontsizeSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontsizeHorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>NoteClientClass</receiver>
   <slot>setFont(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setCodeButtonText(bool)</slot>
  <slot>setFont(int)</slot>
 </slots>
</ui>
